services/oauth_helpers.py

```
@@ -0,0 +1,186 @@
+            - given_name: First name (optional)
+            - family_name: Last name (optional)
+            - picture: Profile picture URL (optional)
+
+    Raises:
+        ValueError: If token cannot be decoded
+    """
+    try:
+        # JWT format: header.payload.signature
+        parts = id_token.split('.')
+        if len(parts) != 3:
+            raise ValueError("Invalid JWT format. Expected 3 parts separated by dots.")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)